### PR TITLE
release 2024-12-02

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,65 @@
...
+
+- [#2522](https://github.com/tigerbeetle/tigerbeetle/pull/2522)
+
+  REPL correctly emits errors when several objects are used as an argument of an operation that
+  only works for a single object, like `get_account_transfers`.
+
+- [#2502](https://github.com/tigerbeetle/tigerbeetle/pull/2502)
+
+  Add randomized integration tests for the Java client.
+
```
Comment (cb22) [inline CHANGELOG.md:39]: Extra newline?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,65 @@
...
+- [#2527](https://github.com/tigerbeetle/tigerbeetle/pull/2527), and
+  [#2487](https://github.com/tigerbeetle/tigerbeetle/pull/2487).
+
+  The Python client!
+
+### Internals
+
+- [#2526](https://github.com/tigerbeetle/tigerbeetle/pull/2526)
+
+  Ignore OOM failures during fuzzing. Fuzzers normally use not that much ram, but, depending on
```
Comment (cb22) [inline CHANGELOG.md]:
```suggestion
  Ignore OOM failures during fuzzing. Fuzzers normally don't use that much ram, but, depending on
```

Reads a little better to me!

Comment (cb22) [reply CHANGELOG.md]: I guess, we should capitalise RAM too since we do below.

Comment (matklad) [reply CHANGELOG.md]: s/RAM/memory !

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,64 @@
...
+  only works for a single object, like `get_account_transfers`.
+
+- [#2502](https://github.com/tigerbeetle/tigerbeetle/pull/2502)
+
+  Add randomized integration tests for the Java client.
+
+### Features
+
+- [#2527](https://github.com/tigerbeetle/tigerbeetle/pull/2527), and
+  [#2487](https://github.com/tigerbeetle/tigerbeetle/pull/2487).
```
Comment (batiati) [inline CHANGELOG.md]: In the bullet points, we usually don't use "and" or the full stop. (plus some inconsistencies)

```suggestion
- [#2527](https://github.com/tigerbeetle/tigerbeetle/pull/2527),
  [#2487](https://github.com/tigerbeetle/tigerbeetle/pull/2487)
```

**Review by batiati — DISMISSED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (matklad) [reply CHANGELOG.md]: Ahh, good catch, copy-paste error from the lead paragraph!

**Review by batiati — APPROVED** [review]

**Pull request merged**